File: js/stack.js
function MyException(message) {
   this.message = message;
   this.name = 'MyException';
}

function Stack(maxLength=0) {
  if (typeof maxLength != 'number' || maxLength < 0) throw new MyException('Incorrect length!');
  this.stack = [];
  this.maxLength = maxLength;

  this.add = function(obj) {
    if (this.stack.length < maxLength) {
    this.stack.push(obj);
    } else {
      console.log('Stack is full now');
    }
    return this;
  };
  this.removeElement = function() {
    this.stack.pop(this.stack.length-1);
    return this;
  };
  this.clear = function() {
    this.stack.length = 0;
    return this;
  }
  return this;
}

module.exports = {
  Stack: Stack,
  MyException: MyException
}

File: python/stack.py
class Stack:
    
    def __init__(self, max_length=0):
        if type(max_length) != int or max_length < 0:
            raise Exception
        self.stack = []
        self.max_length = max_length

    def add(self, obj):     
        if self.max_length > len(self.stack):
            self.stack.append(obj)
        else:
            print('Stack is full')
        return self

    def remove(self):
        try:
            self.stack.pop(len(self.stack)-1)
        except IndexError:
            print('Cannot remove element from stack - Stack is empty')
        return self

    def clear(self):
        self.stack = []
        return self
